bodhi/client/__init__.py

```diff
@@ -440,11 +438,7 @@ def request(update, state, user, password, url, **kwargs):
     client = bindings.BodhiClient(base_url=url, username=user, password=password,
                                   staging=kwargs['staging'])
 
-    try:
-        resp = client.request(update, state)
-    except bindings.UpdateNotFound as exc:
-        raise click.BadParameter(six.text_type(exc), param_hint='UPDATE')
-
+    resp = client.request(update, state)
```

Comment:
I think we want to keep this try/except the way it was, because it lets us give a nice hint to the user that is specific to this command.